src/views/viewer/ChannelView/ChannelView.styles.ts

```diff
@@ -28,31 +29,64 @@ const activeUnderline = css`
 `
 
 export const TitleSection = styled.div`
+  margin-top: -${sizes(6)};
+  display: grid;
+  grid-template-rows: repeat(auto, 3);
+  grid-template-columns: 1fr;
+  grid-row-gap: ${sizes(6)};
+  justify-items: center;
+  width: 100%;
+
+  ${media.sm} {
+    margin-top: 0;
+    grid-template-columns: min-content max-content auto;
+  }
+`
+
+export const CollectorsBox = styled.div`
```

Comment:
I don't see the point of creating this, let's just use [Collectors panel](https://github.com/Joystream/atlas/pull/1948) once it's merged. I would just remove this and not include this in this PR. I know that my PR is blocking you, but I think we could include finished CollectorsBox in a separate PR.